mantidproject/mantid
# Qt3 warnings cleanup group 5

QT3 warning reduction group 5

**To test:**

Check that MantidPlot builds. Changes have been made to:

Integration.cpp
InterpolationDialog.cpp
LabelTool.cpp
LayerDialog.cpp
LineDialog.cpp
LineProfileTool.cpp
LineViewer.cpp
MantidDock.cpp
MantidMatrix.cpp
MantidSampleLogDialog.cpp
MantidSurfacePlotDialog.cpp
1. lots of changes took place around the creation of context menus when clicking on a graph with the label tool.
2. so plot a graph, select the label tool and click on
   1. the X and y axes (or just below them)
   2. a curve line
   3. The background of the graph, and check that the context menus and separator positions in them are sensible.

re #16129



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
